.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+**Replace this text with a summary of the changes in your PR.
+The more detailed you are, the better.**
+
+-----------------
+
+Before submitting the PR make sure the following are checked:
+
+* [ ] Wrote [good commit messages][1].
+* [ ] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
+* [ ] Used the same coding conventions as the rest of the project.
+* [ ] Feature branch is up-to-date with `master` (if not - rebase it).
+* [ ] [Squashed related commits together][2].
```

Comment:
Reorder the list to put relevant point together - e.g. commit message and squash commits